src/components/ReportCard.js

```diff
@@ -70,69 +94,59 @@ const StyledPill = styled.div`
   display: flex;
   justify-content: center;
   align-items: center;
-  width: 69px;
-  height: 28px;
+  width: 70px;
+  height: 30px;
   border-radius: 20px;
-  position: absolute;
-  left: 25px;
-  top: 25px;
-
-  ${minBreakpointQuery.medium`
-    left: 25px;
-    top: 25px;
-  `}
-
-  ${minBreakpointQuery.large`
-    left: 40px;
-    top: 40px;
-  `}
 `;
 
 const StyledImage = styled.img`
+  display: grid;
+  grid-area: 1 / 1 / 1 / 1;
+  position: relative;
+  border-radius: 15px;
   max-height: 100%;
   max-width: 100%;
+  object-fit: contain;
 `;
 
-const StyledHeader = styled.h3`
-  font-family: ${brandFonts.serif};
-  font-size: 25px;
-  font-weight: 700;
-  color: ${brandColours.tertiary};
-  line-height: 35px;
-  margin-bottom: 20px;
+const StyledContentOuter = styled.div`
+  position: relative;
+  display: grid;
+  grid-area: 1 / 1 / 1 / 1;
+  grid-template-rows: repeat(2, 1fr);
+
+  overflow: hidden;
+  z-index: 20;
+  margin: 20px;
 
   ${minBreakpointQuery.medium`
-    font-size: 20px;
+    margin: 40px;
   `}
 
-  ${minBreakpointQuery.large`
-    font-size: 30px;
-    margin-bottom: 40px;
+  ${minBreakpointQuery.smedium`
+    margin: 30px;
   `}
 `;
 
-const StyledLink = styled.a`
-  font-family: ${brandFonts.sans};
-  font-size: 14px;
-  font-weight: 700;
-  line-height: 20px;
-  color: ${brandColours.quinary};
-
-  &:hover {
-    text-decoration: underline;
-  }
+const StyledContentInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
 `;
 
 const ReportCard = ({ reportImage, alt }) => {
   return (
     <StyledReportCard>
-      <StyledOverlay />
       <StyledImage src={reportImage} alt={alt} />
-      <StyledPill>Report</StyledPill>
-      <StyledContent>
-        <StyledHeader>Travel & Covid-19 The Financial Loss States</StyledHeader>
-        <StyledLink>READ POST</StyledLink>
-      </StyledContent>
+      <StyledContentOuter>
+        <StyledPill>Report</StyledPill>
+        <StyledContentInner>
+          <StyledHeader>
+            Travel & Covid-19 The Financial Loss States
+          </StyledHeader>
```

Comment:
Something is going on to create a much bigger space between these two now

![Screenshot 2024-02-14 at 14 27 45](https://github.com/tomkeyes-attachdigital/DiversityTravelHomePage/assets/60260005/237ac4fa-9d32-4da9-8173-33b684c4f942)